### PR TITLE
Update navigation design for user and organisation settings

### PR DESCRIPTION
**Pull request by umpox**
## Overview
This PR implements the updated navigation design for the user and org setting areas.

## Screenshots
| BEFORE             |  AFTER |
:-------------------------:|:-------------------------:
image | image
image | image

## Progress
- [X] Documentation text/screenshots updated (if relevant)
- [x] Approved by a frontend engineer (if touching frontend code)
- [ ] Approved by a backend engineer (if touching backend code)
- [ ] Approved by a designer (if it touches the UI)

Closes https://github.com/sourcegraph/sourcegraph/issues/13376

### CODE REVIEW

**Review by limitedmage — COMMENTED** [review]
Can we add storybook tests for the changed components?

File: client/web/src/components/PageHeader.tsx [hunk 1]
```diff
@@ -27,10 +27,19 @@ interface Props {
     byline?: React.ReactNode
     /** Align additional content (e.g. buttons) alongside the heading */
     actions?: React.ReactNode
+    /** Heading element to use, defaults to h1 */
+    headingElement?: 'h1' | 'h2' | 'h3' | 'h4' | 'h5' | 'h6'
```
Comment (tjkandala) [inline client/web/src/components/PageHeader.tsx:31]: 👍️

**Review by tjkandala — APPROVED** [review]

**Review by AlicjaSuska — CHANGES_REQUESTED** [review]
Thank you for making those changes @umpox! The only thing that is not aligned with design is event log in user settings - it should be in a separate section, by itself.

Also, side note: in the user settings header we display 'name (displayName)' in org settings we only display 'name' in the header. I think we missed it when working on breaders.

**Comment by umpox** [comment]
@AlicjaSuska Made those changes!

image

image

There's some follow up modifications in https://github.com/sourcegraph/sourcegraph/pull/18630 (updates icon colours and adds storybook tests in particular). Happy to merge this and iterate further there

**Review by AlicjaSuska — APPROVED** [review]
Awesome! Thank you for fixing all that @umpox 🎉

**Pull request merged**